src/Commands/Tools/ToolsListCommand.cs

```diff
@@ -12,16 +12,21 @@
 namespace AzureMcp.Commands.Tools;
 
 [HiddenCommand]
-public sealed class ToolsListCommand(ILogger<ToolsListCommand> logger) : BaseCommand()
+public sealed class ToolsListCommand : BaseCommand
```

Comment:
Let's use primary ctors please

Reply:
Ah, can't remember why I cahnged that. I will revert.